From 4d0357b2f1467cd38078ab3cf4e287fe561a4123 Mon Sep 17 00:00:00 2001
From: Paul Davis <paul@linuxaudiosystems.com>
Date: Tue, 9 Mar 2021 09:57:11 -0700
Subject: [PATCH 1/8] quartz: no backing store

We already create NSWindows in "buffered" mode, so this is redundant
---
 gdk/gdkwindow.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gdk/gdkwindow.c b/gdk/gdkwindow.c
index d48751e..65528aa 100644
--- a/gdk/gdkwindow.c
+++ b/gdk/gdkwindow.c
@@ -117,7 +117,9 @@
  * remove a custom clip region.
  */
 
+#ifndef __APPLE__
 #define USE_BACKING_STORE	/* Appears to work on Win32, too, now. */
+#endif
 
 /* This adds a local value to the GdkVisibilityState enum */
 #define GDK_VISIBILITY_NOT_VIEWABLE 3
-- 
2.20.1 (Apple Git-117)

